doc/source/whatsnew/v0.18.0.txt

```diff
@@ -355,7 +355,7 @@ Bug Fixes
 - Regression in ``.clip`` with tz-aware datetimes (:issue:`11838`)
 - Bug in ``date_range`` when the boundaries fell on the frequency (:issue:`11804`)
 - Bug in consistency of passing nested dicts to ``.groupby(...).agg(...)`` (:issue:`9052`)
-
+- Bug in ``read_csv`` when reading from a StringIO in threads (:issue:`11790`)
```

Comment:
this is a bug in ``read_csv`` when reading using multiple threads (not restricted to ``StringIO``)